CDPBatchEditor/CommandArguments/CommandEnumeration.cs

```diff
@@ -104,6 +119,12 @@ public enum CommandEnumeration
         /// Subscribe to parameters #
         /// <example>action=Subscribe --parameters=height,length,mass --domain=Thermal</example>
         /// </summary>
-        Subscribe
+        Subscribe,
+
+        /// <summary>
+        /// Subscribe to parameters #
+        /// <example>action=Subscribe --parameters=height,length,mass --domain=Thermal</example>
+        /// </summary>
+        CreateParameterOverride
```

Comment:
I think in the end implementation for this was not added, so CreateParameterOverride can be removed from here. An impementation of Override is already added in version 4.1.0 (21-02-2025).